### ISSUE TITLE
new markers (star, hexagon...) available in ScatterPlot but not in ColormappedScatterPlot

### ISSUE THREAD

**Issue by fred4ets** [enthought/chaco#425]
Hi there,

New ScatterPlot markers as star, hexagon, hexagon2, pentagon and so on are not available with ColormappedScatterPlot.

For instance, if you run Chaco demo example scatter.py in basic/, it works well if you change line
"marker='circle'" by "marker='star'".
But if you run basic/cmap_scatter.py with the same modification, it displays... nothing.

So my question is : could it be possible to implement these new markers in ColormappedScatterPlot please? 

Thanks in advance,

Cheers,

Debian Linux 64bits unstable, Enthought code from git, Python 2.7, wxPython 3.0.2, Chaco 4.7.1.dev45, Traits 4.7.0.dev94 & TraitsUI 6.1.0.dev22

**Comment by corranwebster** [comment]
Hi, thanks for the report.  I have a suspicion as to where the issue may be.

Can you try setting `render_method = 'bruteforce'` on the colormapped scatterplot to see if that makes a difference in your case?

**Comment by corranwebster** [comment]
Thanks for the feedback - that means that the issue isn't what I thought it might be.

**Comment by fred4ets** [comment]
Any news of this issue, please?

Thanks in advance.

**Comment by fred4ets** [comment]
Hello all,

Any news about this issue?

Regards

**Comment by aaronayres35** [comment]
> Any news about this issue?

Hello @fred4ets, I've just opened #672 which I believe should fix the issue.  I was able to use `marker="star"` on `basic/cmap_scatter.py` locally.
The problem seems to have been enable/chaco being out of sync on what markers work with what methods. The PR is not finished yet.  Ideally it will include a regression test, and I think the solution is unideal. It could be made more robust by not hard coding the specific markers for which we can't call `draw_marker_at_points`. I will update it soon.
In any case, a fix should be made in the near future!

Thank you for reporting this issue!
